ntoskrnl/ex/time.c

```diff
@@ -22,14 +22,95 @@ RTL_TIME_ZONE_INFORMATION ExpTimeZoneInfo;
 ULONG ExpLastTimeZoneBias = -1;
 LARGE_INTEGER ExpTimeZoneBias;
 ULONG ExpAltTimeZoneBias;
-ULONG ExpTimeZoneId;
+ULONG ExpTimeZoneId; // 0 = unknown, 1 = standard time, 2 = daylight time
```

Comment:
Use defines or an enum?

Reply:
@Doug-Lyons Original code was using stuff like `TIME_ZONE_ID_STANDARD` etc. Why did you change this?

Reply:
For only three (3) values, I considered this as not necessary. Thanks.